boards/xtensa/esp32/esp32-core/src/Makefile

```diff
@@ -58,7 +58,6 @@ EXTRA_DISTCLEAN = $(call DELFILE, $(SCRIPTOUT))
 include $(TOPDIR)/boards/Board.mk
 
 $(SCRIPTOUT): $(SCRIPTIN) $(CONFIGFILE)
-	# $(call PREPROCESS, $(SCRIPTIN), $@)
```

Comment:
how does removing a commented out line fix the warning?

Reply:
make[2]: 进入目录“/home/haitao/code/apache/nuttx/boards/xtensa/esp32/esp32-core/src”
# 	@echo "CPP:  /home/haitao/code/apache/nuttx/boards/xtensa/esp32/esp32-core/scripts/esp32.template-> /home/haitao/code/apache/nuttx/boards/xtensa/esp32/esp32-core/scripts/esp32_out.ld"
xtensa-esp32-elf-gcc -E -I. -isystem /home/haitao/code/apache/nuttx/include  -Wno-cpp   /home/haitao/code/apache/nuttx/boards/xtensa/esp32/esp32-core/scripts/esp32.template -o  /home/haitao/code/apache/nuttx/boards/xtensa/esp32/esp32-core/scripts/esp32_out.ld
xtensa-esp32-elf-gcc: warning: /home/haitao/code/apache/nuttx/boards/xtensa/esp32/esp32-core/scripts/esp32.template: linker input file unused because linking not done
xtensa-esp32-elf-gcc -isystem /home/haitao/code/apache/nuttx/include -C -P -x c -E /home/haitao/code/apache/nuttx/boards/xtensa/esp32/esp32-core/scripts/esp32.template -o /home/haitao/code/apache/nuttx/boards/xtensa/esp32/esp32-core/scripts/esp32_out.ld
make[2]: 离开目录“/home/haitao/code/apache/nuttx/boards/xtensa/esp32/esp32-core/src”

Reply:
thank you for explanation